docs/dev/contribute.rst

```diff
@@ -76,22 +76,22 @@ Now you are ready to write your new feature and make changes to the code. To ens
     git commit -m "A short description about what changed"
     git push
 
-Making a pull-request
+Making a pull request
 =====================
 
-While you are working, its helpful to start a pull-request (PR) targeting the :code:`master` branch of :code:`pymeasure/pymeasure`. This will allow you to discuss your feature with other contributors. We encourage you to start this pull-request after your first commit.
+While you are working, its helpful to start a pull request (PR) targeting the :code:`master` branch of :code:`pymeasure/pymeasure`. This will allow you to discuss your feature with other contributors. We encourage you to start this pull-request after your first commit.
```

Comment:
Should we mention the possibilities of a draft (besides to typos in this section)?
```suggestion
While you are working, it is helpful to start a pull request (PR) targeting the :code:`master` branch of :code:`pymeasure/pymeasure`. This will allow you to discuss your feature with other contributors. We encourage you to start this pull request already after your first commit.
You may mark a pull request as a draft, if it is in an early state.
```